mk/docker.mk

```diff
@@ -164,6 +164,7 @@ docker-build-%: $(ARTIFACTS_DIR)
 	    $(EXTRA_BUILD_ARGS) \
 	    --tag $(BUILDIMAGE) \
 	    --file $(DOCKERFILE) .
+	runuser -l svcngcc -c 'mkdir -r $(ARTIFACTS_DIR)'
```

Comment:
This seems like a strange place for this change. Why can this not be done outside of the scope of this makefile? Note that these are used by all users who want to build this project.